src/ExpressionEvaluator/Core/Source/ResultProvider/SystemHelpers.cs

```diff
@@ -0,0 +1,45 @@
+﻿// A few dependencies from System.dll, which we want to avoid when compiling against Desktop Framework 2.0.
```

Comment:
I would change the comment to something like:

// A few dependencies from System.dll...we will avoid referencing the full library to promote portability...